buku

```diff
@@ -2475,10 +2479,10 @@ class BukuDb:
             vi_bm_db_path = '~/Library/Application Support/Vivaldi/Default/Bookmarks'
 
             default_ff_folder = os.path.expanduser('~/Library/Application Support/Firefox')
-            profile = get_firefox_profile_name(default_ff_folder)
-            if profile:
-                ff_bm_db_path = ('~/Library/Application Support/Firefox/'
-                                 '{}/places.sqlite'.format(profile))
+            profiles = firefox_profile or get_firefox_profile_names(default_ff_folder)
```

Comment:
Why only Firefox? Tomorrow another users may request the same for another browser. Why not do it now?